src/inmanta/server/agentmanager.py

```diff
@@ -1285,14 +1283,13 @@ async def _make_agent_config(
 
 [agent_rest_transport]
 port=%(port)s
-host=%(serveradress)s
+host=localhost
```

Comment:
This will only work when listening at least on the loopback ip?

i.e. will this work if you listen explicitly on an interface IP and not 0.0.0.0 ?